src/ui-config/onRampServicesConfig.tsx

```diff
@@ -0,0 +1,23 @@
+import { ReactNode } from 'react';
+import TransakLogo from 'public/icons/onRampServices/transak.svg';
+
+interface MakeLinkParams {
+  cryptoSymbol: string;
+  network: string;
+  walletAddress: string;
+}
+
+interface OnRampServices {
+  name: string;
+  makeLink: ({ cryptoSymbol, network, walletAddress }: MakeLinkParams) => string;
+  icon: ReactNode;
+}
+
+export const onRampServices: OnRampServices[] = [
+  {
+    name: 'Transak',
+    makeLink: ({ cryptoSymbol, network, walletAddress }) =>
+      `${process.env.NEXT_PUBLIC_TRANSAK_APP_URL}/?apiKey=${process.env.NEXT_PUBLIC_TRANSAK_API_KEY}&network=${network}&cryptoCurrencyCode=${cryptoSymbol}&walletAddress=${walletAddress}&disableWalletAddressForm=true`,
```

Comment:
Could you change `network` to `network.split(' ')[0]`

The Polygon network is called `Polygon POS` in the Aave app which breaks this link, but this change will fix it

Reply:
Thanks for pointing this out :)
Done